cognite_toolkit/custom_modules/my_example_module/transformations/first.transformation.yaml

```diff
@@ -3,7 +3,14 @@ name: 'example:first:transformation'
 interval: '{{scheduleHourly}}'
 isPaused: true
 destination: 
-  type: None
+  type: 'assets'
 ignoreNullFields: true
 isPublic: true
-conflictMode: upsert
\ No newline at end of file
+conflictMode: upsert
+authentication:
+  clientId: {{cicd_clientId}}
```

Comment:
I think we need to move these variables up one level in the configs as well then